torch/fx/graph_module.py

```diff
@@ -185,6 +187,11 @@ def recompile(self) -> None:
         """
         self.code = self._graph.python_code(root_module='self')
         cls = type(self)
+
+        def print_full_traceback(exctype, value, tb):
+            traceback.print_exception(exctype, value, tb)
+        sys.excepthook = print_full_traceback
```

Comment:
Is this overriding the exception hook for the remainder of the lifetime of the program? If so, can we instead wrap `forward` with a function that sets this at the beginning and resets it to its original value before returning?

Reply:
Done.